client/state/support/reducer.js

```diff
@@ -17,5 +45,6 @@ export function isSupportUser( state = false, { type } ) {
 }
```

Comment:
should this one persist or not persist? it seemed like there should be a symmetry between these, in which case we could wrap the default `combineReducers()` export in `withoutPersistence()` but if they aren't both alike then you can ignore this

Reply:
Neither should really persist except we want the value of `supportSession` to be relayed from Calypso server to Calypso client. Relevant comment [here](https://github.com/Automattic/wp-calypso/pull/30157/files/5041ca049c7f626bf0b00cbffcc59564045a2bf4#r251596747).